### ISSUE TITLE
Something services screwed up

### ISSUE THREAD

**Issue by eatonphil** [linode/python-linode-api#13]
```python
Traceback (most recent call last):
  File "linode-bootstrap.py", line 131, in <module>
    main()
  File "linode-bootstrap.py", line 127, in main
    args.delete)
  File "linode-bootstrap.py", line 16, in run
    l = client.linode.create_instance(serv, dc)
  File "/Users/peaton/projects/linode/python-linode-api/linode/linode_client.py", line 32, in create_instance
    if isinstance(service, Service) and not 'linode' in service.service_type:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by Dorthu** [comment]
Fixed this in 406e3e62aa95853e1bb1a2bfd3563094416d47ef

**Comment by eatonphil** [comment]
+1
